core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -45,6 +45,7 @@
 import io.trino.sql.tree.NodeRef;
```

Comment:
We need some tests with a mock or any other testing connector

Reply:
This is already done in `TestCreateTableTask`.